allensdk/brain_observatory/behavior/dprime.py

```diff
@@ -0,0 +1,84 @@
+import pandas as pd
+import numpy as np
+from scipy.stats import norm
+
+
+def get_hit_rate(hit=None, miss=None, aborted=None, sliding_window=100):
```

Comment:
Are `hit` and `miss` boolean vectors of length nTrials? one-hot-encoded across hit, miss, aborted?

Reply:
Also, why set the defaults to None if all three are required?